### PR TITLE
Multifactor

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -95,15 +115,85 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
...
+      const {authenticationMethod} = result.token;
+      if(!authenticatedMethods.includes(authenticationMethod)) {
+        authenticatedMethods.push(authenticationMethod);
+      }
+
+      // if `authenticationMethod` is `token-client-registration` then
+      // register token client for it
+      if(authenticationMethod === 'token-client-registration') {
+        // FIXME: get client ID from cookie
+        const clientId = 'FIXME';
```
Comment (mattcollier) [inline lib/index.js:161]: What's the situation here, should there be an issue?

**Review by mattcollier — COMMENTED** [review]
LGTM.

File: lib/index.js (hunk 1 above)
Comment (dlongley) [reply lib/index.js:161]: Yes, there should be an issue to implement an actual cookie for that.

**Review by dlongley — APPROVED** [review]
LGTM. I'm sad about the FIXME for token client registration, but we just haven't gotten to it.

**Pull request merged**